SDK.ipynb

```diff
@@ -78,9 +91,21 @@
    },
    "outputs": [],
    "source": [
-    "import globus_sdk\n",
+    "# fill this line in with the code that you got\n",
+    "auth_code = \"YOUR CODE HERE\"\n",
+    "\n",
+    "# and exchange it for a response object containing your token(s)\n",
+    "token_response = native_auth_client.oauth2_exchange_code_for_tokens(auth_code)\n",
```

Comment:
I think either of those would be great. I agree with only implementing the simplest error handling/recovery for this so it doesn't get so noisy.